Sources/Utilities/KMZendeskChatHandler.swift

```diff
@@ -236,13 +293,25 @@ public class KMZendeskChatHandler : NSObject, JWTAuthenticator {
     }
     
     func processChatLogs(logs: [ChatLog]) {
-        guard lastUserMessageCreatedTime != 0 else { return }
-        let filteredArray = logs.filter{$0.createdTimestamp >= TimeInterval(truncating: lastUserMessageCreatedTime)}
-        processAgentMessage(message: filteredArray)
+        let filteredArray = logs.filter{$0.createdTimestamp >= TimeInterval(truncating: lastSyncTime)}
+        for log in filteredArray {
+            if log.type == .message && log.participant == .agent {
```

Comment:
We can use a switch statement here, with `log.type` and `log.participant` being a tuple.

Reply:
done